homeassistant/components/overkiz/number.py

```diff
@@ -153,9 +207,14 @@ def native_value(self) -> float | None:
 
     async def async_set_native_value(self, value: float) -> None:
         """Set new value."""
-        if self.entity_description.inverted:
-            value = self.native_max_value - value
+        if self.entity_description.set_native_value:
+            await self.entity_description.set_native_value(
+                value, self.executor.async_execute_command
+            )
```

Comment:
```suggestion
            )
            return
```

Then drop the else and outdent below

Reply:
Fixed in 66f2f07f05731acf79895612660d32dbf2de659b.

Reply:
Also moved the inverted value up, does not help right now, but this will be more clear for the future.